backend/src/main/java/ToDoList/backend/model/Lst.java

```diff
@@ -0,0 +1,116 @@
+package TodoList.backend.model;
+
+import javax.persistence.*;

```

Comment:
Please check how to avoid wildcard import. 
https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-